From e3d2570f947ede21072fd5a08c0d4bc40b6688ea Mon Sep 17 00:00:00 2001
From: Sasha Levin <sashal@kernel.org>
Date: Mon, 1 Nov 2021 15:12:02 +0100
Subject: HID: playstation: require multicolor LED functionality

From: Jiri Kosina <jkosina@suse.cz>

[ Upstream commit d7f1f9fec09adc1d77092cb2db0e56e2a4efd262 ]

The driver requires multicolor LED support; express that in Kconfig.

Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Sasha Levin <sashal@kernel.org>
---
 drivers/hid/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hid/Kconfig b/drivers/hid/Kconfig
index 3c33bf572d6d3..d78b1c1fb97e9 100644
--- a/drivers/hid/Kconfig
+++ b/drivers/hid/Kconfig
@@ -868,6 +868,7 @@ config HID_PLANTRONICS
 config HID_PLAYSTATION
 	tristate "PlayStation HID Driver"
 	depends on HID
+	depends on LEDS_CLASS_MULTICOLOR
 	select CRC32
 	select POWER_SUPPLY
 	help
-- 
2.33.0

